src/transaction.ts

```diff
@@ -141,7 +141,7 @@ export interface RunUpdateCallback {
  *     // Error handling omitted.
  *   }
  *
- *   // should be called when finished with the snapshot
+ *   // It is called when the snapshot finishes.
```

Comment:
This should be `It should be called when the snapshot finishes`. Since snapshots don't need to be committed or rolled back, it is up to the user to tell us they're done.